### ISSUE TITLE
Cannot import keys from Jade HWW over USB

### ISSUE THREAD

**Issue by Lobbelt** [cryptoadvance/specter-desktop#1480]
I updated both Specter Desktop (1.7.1) and the Jade firmware (01.30 no radio) to the latest version. When trying to import the keys into Specter Desktop, the screen is stuck at "Detecting...", even though the device is plugged in. Retried multiple times in different USB ports as well. Blockstream Green is perfectly detecting (and unlocking) the device.

It seems to be the same issue referenced in #1251.

**Comment by ikohut** [comment]
Same for me. Specter Desktop (1.7.1) and the Jade firmware (0.1.30) too. No luck to connect to Specter, but Blockstream Green app works well

**Comment by k9ert** [comment]
The thing is that i couldn't reproduce the issue. So more details are needed. Which OS are you using? WHich version of the jade-firmware? If you're using Windows, did you do this:
https://help.blockstream.com/hc/en-us/articles/900005443223-Nothing-happens-when-I-plug-my-Blockstream-Jade-to-my-computer

**Comment by Lobbelt** [comment]
My OS is MacOS Big Sur 11.2. Jade firmware details I mentioned above: 01.30 no radio. 

Blockstream Green detects the Jade so no issue there...

**Comment by evliu** [comment]
Same on macos Monterey. Blockstream Green has no issues, I see it in the system profiler under USB, latest Specter and Jade firmware

**Comment by CoderBoogs** [comment]
I am also having the same issue. When I connect the Jade it stays stuck on detecting

**Comment by k9ert** [comment]
I have only a MacOS catalina-machine. I just tried and it's working there. It's the same machine which is building the App-image so it could potentially be an issue with the AppImage optimized for Catalina and not working on later MacOS versions.

So are you all working with the AppImage or is there someone who can confirm that it's (not) working in a pip-installation on a newer MacOS ?

**Comment by evliu** [comment]
I used the AppImage on my Catalina machine and have the same issue as my two Monterey machines. Both Monterey machines are m1, Catalina is Intel. Issue still persists for me.

**Comment by CoderBoogs** [comment]
I am not sure I am using windows

**Comment by stepansnigirev** [comment]
Jade added a new VID-PID pair to the list of USB devices, so it might be that devices that use these PID:VID were not detected by Specter.
We updated the list in https://github.com/cryptoadvance/specter-desktop/pull/1520 so next release should be able to detect all Jades.

**Comment by Lobbelt** [comment]
Interestingly, the issue fixed itself (?). I did not update the Jade firmware and Specter Desktop is still on 1.7.1 and I was just able to import the Jade's keys.

**Comment by evliu** [comment]
> Jade added a new VID-PID pair to the list of USB devices, so it might be that devices that use these PID:VID were not detected by Specter. We updated the list in #1520 so next release should be able to detect all Jades.

Do you have an idea of which version? I'm on 1.8.1 with jade fw 01.32 and still doesn't detect on any of my 3 macs (both m1 and intel)

**Comment by Lobbelt** [comment]
> > Jade added a new VID-PID pair to the list of USB devices, so it might be that devices that use these PID:VID were not detected by Specter. We updated the list in #1520 so next release should be able to detect all Jades.
> 
> Do you have an idea of which version? I'm on 1.8.1 with jade fw 01.32 and still doesn't detect on any of my 3 macs (both m1 and intel)

Do you have any other wallet software running at the same time (e.g. Blockstream Green or Electrum)? I have noticed that that causes issues too, so close down that software as well.

**Comment by evliu** [comment]
Nope, direct usb-c connection, I see it in system profile, no other USB devices connected or other Bitcoin software except Specter. Works totally fine in Green and when I do an `ls` on the /dev/tty/USB or whatever the command is. Same for all computers. I saw in Bitcoin hardware repo there has been a bit of new Jade PRs merged in

**Comment by k9ert** [comment]
> Do you have an idea of which version? 
Just to confirm: yes, it's `v1.8.1`

**Comment by evliu** [comment]
I know this is closed but still haven't been able to connect w/ v1.8.1, fw 01.33, but i'm able to use https://github.com/bitcoin-core/HWI 
when i enumerate, I do get:

``` 
{
    "type": "jade",
    "model": "jade",
    "path": "/dev/cu.usbserial-0224A7E3",
    "needs_pin_sent": false,
    "needs_passphrase_sent": false,
    "fingerprint": "..."
  }
  ```

from there, i'm able to use hwi to grab xpubs, descriptors, sign transactions, etc.

**Comment by evliu** [comment]
@k9ert 
Found the issue. hwi-2.0.2 doesn't detect my jade (i think it has newer hardware) but hwi-2.1.0 does. I retrofitted specter-desktop with the newer version (which required the new pyseriel, hwi, and I used python 3.10)

I'm guessing this fix won't get in for awhile because this will require quite a bit of changes. hwi-2.1.0 has some regression issues in hwi, but i'm glad I got down to the bottom of it.

<img width="633" alt="image" src="https://user-images.githubusercontent.com/1699644/159798399-2e925ac4-db73-45e8-a9a9-f44bdf6bc79b.png">
